weaviate/connection/rest.go

```diff
@@ -53,6 +54,38 @@ func NewConnection(scheme string, host string, httpClient *http.Client, headers
 	return connection
 }
 
+// WaitForWeaviate waits until weaviate is started up and ready
+// expects weaviat at localhost:8080
+func (con *Connection) WaitForWeaviate(startup_period int) error {
+	if startup_period < 0 {
+		return errors.New("'startup_period' needs to be an integer larger than zero")
+	}
+
+	for i := 0; i < startup_period; i++ {
+		ctx, cancelFunc := context.WithTimeout(context.Background(), time.Second*3)
```

Comment:
this should be one second, otherwise this would wait for up to 3 seconds ( I think)